e2e/tests/server.js

```diff
@@ -23,20 +23,6 @@ before(
         })
 );
 
-before(() => {
```

Comment:
Because I had weird issues when some tests were trying to click buttons hidden under notifications. You may have noticed that the chrome driver is not started with the `start-maximized` option to ensure the largest screen possible. 

Another option would have been to implement some custom webdriver code which would allow to scroll until an element is clickable but it required more time to implement